### ISSUE TITLE
acc = 50

### ISSUE THREAD

**Issue by shui-tian-ju-shi** [mapooon/SelfBlendedImages#51]
I trained a model by myself. The accuracy (acc) on the validation set is always only 50. When I test with the pre-trained model released by the author, the acc is also only 50. I only changed the relevant parameters in the code. Can you provide some suggestions?

I may not have completely downloaded the correct version of the package. Could the problem be related to this?

**Comment by shui-tian-ju-shi** [comment]
I found through debugging that the mask value generated for mixing is 0 . I don't know why this is.
